src/Components/SongSelector/SongSelector.js

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+import { searchSong } from "Services/Spotify/spotifyAPI";
+import useSpotifyAuthData from "Store/selectors/useSpotifyAuthData";
+import SearchFormView from "./SearchFormView";
+import SearchResultsView from "./SearchResultsView";
+import SelectedSongView from "./SelectedSongView";
+import styles from "./SongSelector.module.css";
+
+const SongSelector = function () {
+  const { access_token } = useSpotifyAuthData();
+
+  const [searchResults, setSearchResults] = useState([]);
+  const [selectedSong, setSelectedSong] = useState(null);
+  const [songConfirmed, setSongConfirmed] = useState(false);
+
+  const handleSearch = (query) => {
+    searchSong(access_token, query).then((data) => {
```

Comment:
Maybe we could here if the query is not null, otherwise we get an error the spotifyAPI